### ISSUE TITLE
"RowanDebugger does not understand #browseClass" from double click on frame selection (3.0.41)

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#238]
![image](https://user-images.githubusercontent.com/623951/47036314-19b9a680-d131-11e8-8f32-645b1f43e4a5.png)

stack dump:
```
************************** Dolphin Virtual Machine Dump Report ***************************

10:48:02 AM, 10/16/2018: RowanDebugger does not understand #browseClass

*----> VM Context <----*
Process: {1D660210:size 789 words, suspended frame 1D660DFD, priority 6, callbacks 2
last failure 2:nil, FP control 80017, thread 00000000}
Active Method: SessionManager>>logError:
IP: 021FD5B9 (9)
SP: 1D660FB4
BP: 1D660F88 (847)
ActiveFrame: {1D660F8C: cf 1D660F6D, sp 1D660FA4, bp 1D660F88, ip 5, JadeSessionManager(SessionManager)>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a MessageNotUnderstood


New Method: VMLibrary>>dump:path:stackDepth:walkbackDepth:
Message Selector: dump:path:stackDepth:walkbackDepth:

*----> Stack Back Trace <----*
{1D660F8C: cf 1D660F6D, sp 1D660FA4, bp 1D660F88, ip 5, JadeSessionManager(SessionManager)>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a MessageNotUnderstood

{1D660F6C: cf 1D660F4D, sp 1D660F80, bp 1D660F68, ip 43, [] in GsRuntimeError(GsError)>>reportError}
	receiver: a GsRuntimeError
	arg[0]: a MessageNotUnderstood

{1D660F4C: cf 1D660F1D, sp 1D660F60, bp 1D660F40, ip 40, [] in MessageNotUnderstood(Exception)>>_evaluateHandler:in:}
	receiver: a MessageNotUnderstood

{1D660F1C: cf 1D660EFD, sp 1D660F30, bp 1D660F18, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 52 in Exception>>_evaluateHandler:in:

{1D660EFC: cf 1D660ED9, sp 1D660F10, bp 1D660EF4, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 21 in Exception>>_evaluateHandler:in:
	arg[0]: [] @ 52 in Exception>>_evaluateHandler:in:
	stack temp[0]: nil

{1D660ED8: cf 1D660EA1, sp 1D660EEC, bp 1D660EBC, ip 61, MessageNotUnderstood(Exception)>>_evaluateHandler:in:}
	receiver: a MessageNotUnderstood
	arg[0]: [] @ 29 in GsError>>reportError
	arg[1]: a ExceptionHandler
	stack temp[0]: nil
	stack temp[1]: a ExceptionHandler
	stack temp[2]: nil
	stack temp[3]: nil
	stack temp[4]: a Process('Main' base 1D660000 [ACTIVE] in SessionManager>>logError: sp=00000000 ip=4 list=02640010)

{1D660EA0: cf 1D660E6D, sp 1D660EB4, bp 1D660E88, ip 43, MessageNotUnderstood(Exception)>>_propagateFrom:}
	receiver: a MessageNotUnderstood
	arg[0]: a ExceptionHandler
	stack temp[0]: a ExceptionHandler
	stack temp[1]: a ExceptionHandler
	stack temp[2]: nil
	stack temp[3]: a Process('Main' base 1D660000 [ACTIVE] in SessionManager>>logError: sp=00000000 ip=4 list=02640010)
	stack temp[4]: [] @ 29 in GsError>>reportError

{1D660E6C: cf 1D660E4D, sp 1D660E80, bp 1D660E68, ip 6, MessageNotUnderstood(Exception)>>_propagate}
	receiver: a MessageNotUnderstood
	stack temp[0]: nil

{1D660E4C: cf 1D660E31, sp 1D660E60, bp 1D660E4C, ip 13, MessageNotUnderstood(Exception)>>signal}
	receiver: a MessageNotUnderstood

{1D660E30: cf 1D660E0D, sp 1D660E44, bp 1D660E28, ip 12, MessageNotUnderstood class>>receiver:message:}
	receiver: MessageNotUnderstood
	arg[0]: a RowanDebugger
	arg[1]: Message selector: browseClass arguments: #()

{1D660E0C: cf 1D660DED, sp 1D660E20, bp 1D660E08, ip 4, RowanDebugger(Object)>>doesNotUnderstand:}
	receiver: a RowanDebugger
	arg[0]: Message selector: browseClass arguments: #()

{1D660DEC: cf 1D660DC9, sp 1D660E00, bp 1D660DE4, ip 13, EventMessageSend>>forwardTo:withArguments:}
	receiver: a EventMessageSend
	arg[0]: a RowanDebugger
	arg[1]: a Array

{1D660DC8: cf 1D660DAD, sp 1D660DDC, bp 1D660DC8, ip 5, EventMessageSend>>value}
	receiver: a EventMessageSend

{1D660DAC: cf 1D660D85, sp 1D660DC0, bp 1D660DA8, ip 11, [] in MessageSequenceAbstract>>value}
	receiver: nil
	arg[0]: a EventMessageSend

{1D660D84: cf 1D660D55, sp 1D660DA0, bp 1D660D70, ip 30, EventMessageSequence>>messagesDo:}
	receiver: a EventMessageSequence
	arg[0]: [] @ 9 in MessageSequenceAbstract>>value
	stack temp[0]: a DeadObject
	stack temp[1]: 1
	stack temp[2]: a RowanDebugger
	stack temp[3]: a EventMessageSend

{1D660D54: cf 1D660D39, sp 1D660D68, bp 1D660D54, ip 15, EventMessageSequence(MessageSequenceAbstract)>>value}
	receiver: a EventMessageSequence
	env temp[0]: nil

{1D660D38: cf 1D660D19, sp 1D660D4C, bp 1D660D34, ip 8, EventsCollection>>triggerEvent:}
	receiver: a EventsCollection
	arg[0]: actionPerformed

{1D660D18: cf 1D660CF9, sp 1D660D2C, bp 1D660D14, ip 4, ListPresenter(Object)>>trigger:}
	receiver: a ListPresenter
	arg[0]: actionPerformed

{1D660CF8: cf 1D660CD5, sp 1D660D0C, bp 1D660CF8, ip 13, [] in ListBox(View)>>onActionPerformed}
	receiver: a ListBox

{1D660CD4: cf 1D660CB5, sp 1D660CE8, bp 1D660CD0, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 18 in Cursor>>showWhile:

{1D660CB4: cf 1D660C91, sp 1D660CC8, bp 1D660CAC, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 10 in View>>onActionPerformed
	arg[0]: [] @ 18 in Cursor>>showWhile:
	stack temp[0]: nil

{1D660C90: cf 1D660C69, sp 1D660CA4, bp 1D660C84, ip 27, Cursor>>showWhile:}
	receiver: a Cursor
	arg[0]: [] @ 10 in View>>onActionPerformed
	stack temp[0]: nil
	stack temp[1]: a ExternalHandle(00010003)

{1D660C68: cf 1D660C4D, sp 1D660C7C, bp 1D660C68, ip 15, ListBox(View)>>onActionPerformed}
	receiver: a ListBox

{1D660C4C: cf 1D660C31, sp 1D660C60, bp 1D660C4C, ip 6, ListBox(ListControlView)>>onActionPerformed}
	receiver: a ListBox

{1D660C30: cf 1D660C15, sp 1D660C44, bp 1D660C30, ip 3, ListPresenter(Presenter)>>onActionPerformed}
	receiver: a ListPresenter

{1D660C14: cf 1D660BF9, sp 1D660C28, bp 1D660C14, ip 3, ListBox(View)>>performAction}
	receiver: a ListBox

{1D660BF8: cf 1D660BDD, sp 1D660C0C, bp 1D660BF8, ip 3, ListPresenter(Presenter)>>performAction}
	receiver: a ListPresenter

{1D660BDC: cf 1D660BB9, sp 1D660BF0, bp 1D660BD4, ip 7, ListBox(BasicListAbstract)>>onLeftButtonDoubleClicked:}
	receiver: a ListBox
	arg[0]: a MouseEvent
	stack temp[0]: 0

{1D660BB8: cf 1D660B99, sp 1D660BCC, bp 1D660BB4, ip 4, ListPresenter(Presenter)>>onLeftButtonDoubleClicked:}
	receiver: a ListPresenter
	arg[0]: a MouseEvent

{1D660B98: cf 1D660B71, sp 1D660BAC, bp 1D660B8C, ip 9, ListBox(View)>>wmLButtonDblClk:wParam:lParam:}
	receiver: a ListBox
	arg[0]: 515
	arg[1]: 1
	arg[2]: 12058811

{1D660B70: cf 1D660B41, sp 1D660B84, bp 1D660B5C, ip 23, ListBox(View)>>dispatchMessage:wParam:lParam:}
	receiver: a ListBox
	arg[0]: 515
	arg[1]: 1
	arg[2]: 12058811
	stack temp[0]: nil
	stack temp[1]: wmLButtonDblClk:wParam:lParam:

{1D660B40: cf 1D660B0D, sp 1D660B54, bp 1D660B30, ip 69, [] in InputState>>wndProc:message:wParam:lParam:cookie:}
	receiver: nil

{1D660B0C: cf 1D660AE5, sp 1D660B20, bp 1D660B08, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 79 in InputState>>wndProc:message:wParam:lParam:cookie:

{1D660AE4: cf 1D660AA5, sp 1D660B00, bp 1D660ACC, ip 84, InputState>>wndProc:message:wParam:lParam:cookie:}
	receiver: a InputState
	arg[0]: 460120
	arg[1]: 515
	arg[2]: 1
	arg[3]: 12058811
	arg[4]: 850892
	stack temp[0]: a ListBox

{1D660AA4: cf 1D660A85, sp 1D660AC4, bp 1D660AA0, ip 6, ShellView(View)>>isDialogMessage:}
	receiver: a ShellView
	arg[0]: a MSG

{1D660A84: cf 1D660A65, sp 1D660A98, bp 1D660A80, ip 5, ShellView>>preTranslateMouseInput:}
	receiver: a ShellView
	arg[0]: a MSG

{1D660A64: cf 1D660A45, sp 1D660A78, bp 1D660A60, ip 13, ShellView(View)>>preTranslateMessage:}
	receiver: a ShellView
	arg[0]: a MSG

{1D660A44: cf 1D660A1D, sp 1D660A58, bp 1D660A38, ip 17, InputState>>preTranslateMessage:}
	receiver: a InputState
	arg[0]: a MSG
	stack temp[0]: 853310
	stack temp[1]: a ShellView

{1D660A1C: cf 1D6609FD, sp 1D660A30, bp 1D660A18, ip 19, InputState>>pumpMessage:}
	receiver: a InputState
	arg[0]: a MSG

{1D6609FC: cf 1D6609D1, sp 1D660A10, bp 1D6609EC, ip 18, InputState>>loopWhile:}
	receiver: a InputState
	arg[0]: [] @ 14 in JadeDebugger>>answer
	stack temp[0]: a MSG
	stack temp[1]: true
	stack temp[2]: nil

{1D6609D0: cf 1D6609B5, sp 1D6609E4, bp 1D6609D0, ip 18, RowanDebugger(JadeDebugger)>>answer}
	receiver: a RowanDebugger

{1D6609B4: cf 1D66098D, sp 1D6609C8, bp 1D6609A8, ip 13, RowanDebugger class(JadeDebugger class)>>openOn:message:terminateOnClose:}
	receiver: RowanDebugger
	arg[0]: a GsProcess
	arg[1]: 'a FileDoesNotExistException occurred (error 2710)'
	arg[2]: true

{1D66098C: cf 1D66096D, sp 1D6609A0, bp 1D660988, ip 9, RowanDebugger class(JadeDebugger class)>>openDebuggerOnException:}
	receiver: RowanDebugger
	arg[0]: a GsRuntimeError

{1D66096C: cf 1D660945, sp 1D660980, bp 1D660960, ip 66, RowanDebugger class(JadeDebugger class)>>reportError:}
	receiver: RowanDebugger
	arg[0]: a GsRuntimeError
	stack temp[0]: nil
	stack temp[1]: debug

{1D660944: cf 1D660929, sp 1D660958, bp 1D660944, ip 5, GsRuntimeError(GsError)>>debugError}
	receiver: a GsRuntimeError

{1D660928: cf 1D66090D, sp 1D66093C, bp 1D660928, ip 20, [] in GsRuntimeError(GsError)>>reportError}
	receiver: a GsRuntimeError

{1D66090C: cf 1D6608F1, sp 1D660920, bp 1D66090C, ip 13, ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry}
	receiver: a ExceptionHandler

{1D6608F0: cf 1D6608C9, sp 1D660904, bp 1D6608EC, ip 22, [] in ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler

{1D6608C8: cf 1D6608A9, sp 1D6608DC, bp 1D6608C4, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:

{1D6608A8: cf 1D660885, sp 1D6608BC, bp 1D6608A0, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 17 in ExceptionHandlerAbstract>>try:
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:
	stack temp[0]: nil

{1D660884: cf 1D66085D, sp 1D660898, bp 1D660878, ip 44, ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler
	arg[0]: [] @ 18 in GsError>>reportError
	stack temp[0]: a ExceptionHandler
	stack temp[1]: a Process('Main' base 1D660000 [ACTIVE] in SessionManager>>logError: sp=00000000 ip=4 list=02640010)
	env temp[0]: nil

{1D66085C: cf 1D660839, sp 1D660870, bp 1D660854, ip 7, BlockClosure>>on:do:}
	receiver: [] @ 18 in GsError>>reportError
	arg[0]: Error
	arg[1]: [] @ 29 in GsError>>reportError

{1D660838: cf 1D66081D, sp 1D66084C, bp 1D660838, ip 66, GsRuntimeError(GsError)>>reportError}
	receiver: a GsRuntimeError

{1D66081C: cf 1D660801, sp 1D660830, bp 1D66081C, ip 2, GsRuntimeError(GsError)>>defaultAction}
	receiver: a GsRuntimeError

{1D660800: cf 1D6607CD, sp 1D660814, bp 1D6607E8, ip 55, GsRuntimeError(Exception)>>_propagateFrom:}
	receiver: a GsRuntimeError
	arg[0]: a ExceptionHandler
	stack temp[0]: nil
	stack temp[1]: a ExceptionHandler
	stack temp[2]: nil
	stack temp[3]: a Process('Main' base 1D660000 [ACTIVE] in SessionManager>>logError: sp=00000000 ip=4 list=02640010)
	stack temp[4]: nil

{1D6607CC: cf 1D6607AD, sp 1D6607E0, bp 1D6607C8, ip 6, GsRuntimeError(Exception)>>_propagate}
	receiver: a GsRuntimeError
	stack temp[0]: nil

{1D6607AC: cf 1D660791, sp 1D6607C0, bp 1D6607AC, ip 13, GsRuntimeError(Exception)>>signal}
	receiver: a GsRuntimeError

{1D660790: cf 1D660775, sp 1D6607A4, bp 1D660790, ip 5, GsRuntimeError(GsError)>>signal}
	receiver: a GsRuntimeError

{1D660774: cf 1D660749, sp 1D660788, bp 1D660764, ip 17, GciSession>>withExplanation:do:}
	receiver: a GciSession
	arg[0]: 'jadeServer\0D\\0A\\09\perform: #'debugTestNamed:of:'\0D\\0A\\09\withAll: (Array\0D\\0A\\09\\09\with: #testTone...'
	arg[1]: [] @ 73 in GciSession>>send:to:withAll:
	stack temp[0]: a GsRuntimeError
	stack temp[1]: a GsRuntimeError

{1D660748: cf 1D66071D, sp 1D66075C, bp 1D660738, ip 81, GciSession>>send:to:withAll:}
	receiver: a GciSession
	arg[0]: debugTestNamed:of:
	arg[1]: a OopType64
	arg[2]: a Array
	stack temp[0]: a WriteStream

{1D66071C: cf 1D6606F5, sp 1D660730, bp 1D660710, ip 13, GciSession>>serverPerform:withArguments:}
	receiver: a GciSession
	arg[0]: debugTestNamed:of:
	arg[1]: a Array
	stack temp[0]: nil

{1D6606F4: cf 1D6606CD, sp 1D660708, bp 1D6606E8, ip 7, GciSession>>serverPerform:with:with:}
	receiver: a GciSession
	arg[0]: debugTestNamed:of:
	arg[1]: testTonelWriter_A
	arg[2]: 'RwProjectWriterTest'

{1D6606CC: cf 1D6606B1, sp 1D6606E0, bp 1D6606CC, ip 28, JadeSUnitPresenter>>debug}
	receiver: a JadeSUnitPresenter

{1D6606B0: cf 1D660691, sp 1D6606C4, bp 1D6606AC, ip 4, Symbol>>forwardTo:}
	receiver: debug
	arg[0]: a JadeSUnitPresenter

{1D660690: cf 1D660671, sp 1D6606A4, bp 1D66068C, ip 4, CommandDescription>>performAgainst:}
	receiver: a CommandDescription
	arg[0]: a JadeSUnitPresenter

{1D660670: cf 1D66064D, sp 1D660684, bp 1D660670, ip 14, [] in Command>>value}
	receiver: a Command

{1D66064C: cf 1D66062D, sp 1D660660, bp 1D660648, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 17 in Command>>value

{1D66062C: cf 1D660609, sp 1D660640, bp 1D660624, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 11 in Command>>value
	arg[0]: [] @ 17 in Command>>value
	stack temp[0]: nil

{1D660608: cf 1D6605ED, sp 1D66061C, bp 1D660608, ip 22, Command>>value}
	receiver: a Command

{1D6605EC: cf 1D6605CD, sp 1D660600, bp 1D6605E8, ip 30, ShellView>>performCommand:}
	receiver: a ShellView
	arg[0]: a Command

{1D6605CC: cf 1D6605AD, sp 1D6605E0, bp 1D6605C8, ip 4, JadeSUnitBrowser(Shell)>>performCommand:}
	receiver: a JadeSUnitBrowser
	arg[0]: a Command

{1D6605AC: cf 1D660585, sp 1D6605C0, bp 1D6605A0, ip 10, CommandQuery>>perform}
	receiver: a CommandQuery
	stack temp[0]: a Command
	stack temp[1]: nil
	stack temp[2]: a JadeSUnitBrowser

{1D660584: cf 1D660561, sp 1D660598, bp 1D66057C, ip 8, DelegatingCommandPolicy(CommandPolicy)>>route:}
	receiver: a DelegatingCommandPolicy
	arg[0]: a CommandDescription
	stack temp[0]: a CommandQuery

{1D660560: cf 1D660539, sp 1D660574, bp 1D66055C, ip 15, [] in MultipleSelectionListView(View)>>onCommand:}
	receiver: a MultipleSelectionListView

{1D660538: cf 1D660519, sp 1D66054C, bp 1D660534, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 18 in Cursor>>showWhile:

{1D660518: cf 1D6604F5, sp 1D66052C, bp 1D660510, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 11 in View>>onCommand:
	arg[0]: [] @ 18 in Cursor>>showWhile:
	stack temp[0]: nil

{1D6604F4: cf 1D6604CD, sp 1D660508, bp 1D6604E8, ip 27, Cursor>>showWhile:}
	receiver: a Cursor
	arg[0]: [] @ 11 in View>>onCommand:
	stack temp[0]: nil
	stack temp[1]: a ExternalHandle(00010003)

{1D6604CC: cf 1D6604A9, sp 1D6604E0, bp 1D6604C4, ip 17, MultipleSelectionListView(View)>>onCommand:}
	receiver: a MultipleSelectionListView
	arg[0]: a CommandDescription
	stack temp[0]: nil

{1D6604A8: cf 1D660475, sp 1D6604BC, bp 1D660490, ip 19, MultipleSelectionListView(View)>>wmCommand:wParam:lParam:}
	receiver: a MultipleSelectionListView
	arg[0]: 273
	arg[1]: 7699
	arg[2]: 0
	stack temp[0]: 7699
	stack temp[1]: a CommandDescription
	stack temp[2]: nil

{1D660474: cf 1D660445, sp 1D660488, bp 1D660460, ip 23, MultipleSelectionListView(View)>>dispatchMessage:wParam:lParam:}
	receiver: a MultipleSelectionListView
	arg[0]: 273
	arg[1]: 7699
	arg[2]: 0
	stack temp[0]: nil
	stack temp[1]: wmCommand:wParam:lParam:

{1D660444: cf 1D660411, sp 1D660458, bp 1D660434, ip 69, [] in InputState>>wndProc:message:wParam:lParam:cookie:}
	receiver: nil

{1D660410: cf 1D6603E9, sp 1D660424, bp 1D66040C, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 79 in InputState>>wndProc:message:wParam:lParam:cookie:

{1D6603E8: cf 1D6603AD, sp 1D660404, bp 1D6603D0, ip 84, InputState>>wndProc:message:wParam:lParam:cookie:}
	receiver: a InputState
	arg[0]: 66852
	arg[1]: 273
	arg[2]: 7699
	arg[3]: 0
	arg[4]: 851284
	stack temp[0]: a MultipleSelectionListView

{1D6603AC: cf 1D66038D, sp 1D6603C8, bp 1D6603A8, ip 28, InputState>>pumpMessage:}
	receiver: a InputState
	arg[0]: a MSG

{1D66038C: cf 1D660361, sp 1D6603A0, bp 1D66037C, ip 18, InputState>>loopWhile:}
	receiver: a InputState
	arg[0]: [] @ 9 in InputState>>mainLoop
	stack temp[0]: a MSG
	stack temp[1]: true
	stack temp[2]: nil

{1D660360: cf 1D660345, sp 1D660374, bp 1D660360, ip 15, InputState>>mainLoop}
	receiver: a InputState

{1D660344: cf 1D660329, sp 1D660358, bp 1D660344, ip 14, [] in InputState>>forkMain}
	receiver: a InputState

{1D660328: cf 1D66030D, sp 1D66033C, bp 1D660328, ip 13, ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry}
	receiver: a ExceptionHandler

{1D66030C: cf 1D6602E5, sp 1D660320, bp 1D660308, ip 22, [] in ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler

{1D6602E4: cf 1D6602C5, sp 1D6602F8, bp 1D6602E0, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:

{1D6602C4: cf 1D6602A1, sp 1D6602D8, bp 1D6602BC, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 17 in ExceptionHandlerAbstract>>try:
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:
	stack temp[0]: nil

{1D6602A0: cf 1D660279, sp 1D6602B4, bp 1D660294, ip 44, ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler
	arg[0]: [] @ 10 in InputState>>forkMain
	stack temp[0]: nil
	stack temp[1]: a Process('Main' base 1D660000 [ACTIVE] in SessionManager>>logError: sp=00000000 ip=4 list=02640010)
	env temp[0]: nil

{1D660278: cf 1D660255, sp 1D66028C, bp 1D660270, ip 7, BlockClosure>>on:do:}
	receiver: [] @ 10 in InputState>>forkMain
	arg[0]: ProcessTermination
	arg[1]: [] @ 13 in BlockClosure>>newProcess

{1D660254: cf 00000001, sp 1D660268, bp 1D660254, ip 17, [] in BlockClosure>>newProcess}
	receiver: [] @ 10 in InputState>>forkMain

<Bottom of stack>

***** End of dump *****
```

**Comment by ericwinger** [comment]
Double click now opens frame method. Fixed in 3.0.47.

**Comment by LisaAlmarode** [comment]
If you double click on a block context or the 'Executed Code' frame you get a walkback.

a ArgumentTypeError occurred (error 2094), reason:rtErrBadArgKind, expected a CharacterCollection
ArgumentTypeError (AbstractException) >> _signalWith: @6 line 25
ArgumentTypeError (AbstractException) >> signal @2 line 47
UndefinedObject (Object) >> _error:args: @15 line 11
UndefinedObject (Object) >> _errorExpectedClass: @2 line 5
UndefinedObject (Object) >> _validateClass: @4 line 6
Symbol class >> _existingWithAll: @2 line 10
SymbolList >> resolveSymbol: @4 line 9
SymbolList >> objectNamed: @2 line 8
RwGsImage class >> objectNamed: @3 line 6
RwGsPlatform >> globalNamed: @3 line 5
Rowan class >> globalNamed: @3 line 5
RowanClassService >> classFromName @2 line 3
RowanClassService >> updateClass @2 line 7
RowanClassService >> update @2 line 3
[] in  ExecBlock1 (JadeServer) >> updateFromSton: @6 line 11
Array (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> updateFromSton: @7 line 6
GsNMethod class >> _gsReturnToC @1 line 1

**Comment by ericwinger** [comment]
Fixed in 3.0.49
